### PR TITLE
Avatar verify-failed banner should click to appropriate app

### PR DESCRIPTION
**Pull request by SimonWalton-HiFi**
https://highfidelity.atlassian.net/browse/BUGZ-206 (in part).

Clicking on the banner (actually a local entity with a script) should open the appropriate app, depending whether or not simplifiedUI is running.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19507/HighFidelity-Beta-PR15574-54a76e8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19507/HighFidelity-Beta-Interface-PR15574-54a76e8.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19507/HighFidelity-Beta-PR15574-54a76e8.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19507/HighFidelity-Quest-Beta-PR15574-54a76e8.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19507/HighFidelity-Beta-PR15574-54a76e8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19507/HighFidelity-Beta-Interface-PR15574-54a76e8.exe

File: scripts/system/clickToAvatarApp.js [hunk 1]
```diff
@@ -1,7 +1,17 @@
 (function () {
     var tablet = Tablet.getTablet("com.highfidelity.interface.tablet.system");
     this.clickDownOnEntity = function (entityID, mouseEvent) {
-        tablet.loadQMLSource("hifi/AvatarApp.qml");
+        var runningSimplified = false;
```
Comment (hyperlogic) [inline scripts/system/clickToAvatarApp.js]: Replace the for loop with Array.prototype.find, it's less error prone and more clearly shows intent.

    var scripts = ScriptDiscoveryService.getRunning();
    var runningSimplified = scripts.find(function (elem) {
        return elem.name === "simplifiedUI.js";
    });

or if you want to guarantee that runningSimplified is a boolean value.

    var scripts = ScriptDiscoveryService.getRunning();
    var runningSimplified = scripts.findIndex(function (elem) {
        return elem.name === "simplifiedUI.js";
    }) != -1;

**Comment by howard-stearns** [comment]
We don't know exactly what our design will be for simplified ui support (e.g., is it a supported thing with a stable name, such that other system behavior can rely on it), nor on what the avatar chooser will be (e.g., will we have an hifi/AvatarApp.qml AND a hifi/simplifiedUI/avatarApp/AvatarApp.qml, or just one redesigned AvatarApp.qml?).

And there's no need to guess about this (or force the questions) right now, as the verification banner is out from m1 anyway.

**Pull request closed without merging**